cmd/bosun/web/static/js/graph.ts

```diff
@@ -466,7 +469,12 @@ bosunControllers.controller('GraphCtrl', ['$scope', '$http', '$location', '$rout
 		});
 		return request;
 	}
-	$scope.Query = function() {
+		$scope.keydown = function ($event: any) {
```

Comment:
I'm curious why only certain fields attach this event, and others do not. Would it work to attach the keydown event to the entire view and handle it no matter where they are focused?